apiary.apib

```diff
@@ -6197,7 +6197,406 @@ curl -H "content-type: application/json" -H "accept:application/json" -X PUT -d
 + Response 400
 
     [2078][]
+    
+
+## Get [/v1/review/report/count-by-rating/{BusinessId}{?api_key}]
+Business review(s) from BirdEye platform can be fetched using following filter in request payload:
+
++ Parameters
+
+    + BusinessId (string, required) ... the Business ID
+    + api_key (string, required) ... the API key
+
+### Get Reviews BY Count [POST]
+
+#### Request
+
+<center><b>Example curl request</b></center>
+
+    curl -XPOST -H "Content-type: application/json" -d '{"businessNumbers": [145308764185002,145308769721320],"sourceAlias": ["google","citysearch"],"fromDate": "04/01/2013","toDate": "05/3/2015"}' 'https://api.birdeye.com/resources/v1/review/report/count-by-rating/145308769721320?api_key=92bcd6e0-c102-43fd-8a67-1a7be5258451'
```

Comment:
Should be like curl -H "content-type: application/json" -H "accept:application/json" -X POST to keep uniformity with existing documentation.

Reply:
Done